### ISSUE TITLE
Add a get_session() utility function

### ISSUE THREAD

**Issue by jesper-friis** [EMMC-ASBL/otelib#87]
Add a `get_session()` utility function and replace all current occurences of 

```python
AttrDict(**json.loads(raw_session))
```

with `get_session(raw_session)`, where `raw_session` is what is returned by the `fetch()`, `get()` and `initialize()` methods.

This makes the code easier to read and makes it much easier to change the return value of the `fetch()`, `get()` and `initialize()` methods as suggested in issue #86. 

If issue #86 is rejected, this utility function would still be improve the user-friendliness (although not as much as issue #86).

**Comment by CasperWA** [comment]
I don't think this is relevant any longer? Issue #86 is being implemented by #134.
